common-travel.lic

```diff
@@ -69,6 +73,31 @@ module DRCT
     fput("offer #{amount}") if amount
   end
 
+  # Visit a merchant and ask for an item.
+  # Usually this is for bundling ropes or gem pouches.
+  def ask_for_item?(room, name, item)
+    walk_to(room)
+
+    success_patterns = [
+      /hands you/
+    ]
+
+    failure_patterns = [
+      /does not seem to know anything about that/,
+      /All I know about/,
+      /To whom are you speaking/,
+      /Usage: ASK/
+    ]
+
+    case DRC.bput("ask #{name} for #{item}", *success_patterns, *failure_patterns)
```

Comment:
I still prefer the `==` syntax

```suggestion
    DRC.bput("ask #{name} for #{item}", *success_patterns, *failure_patterns) == 'hands you
```

But it's not a big deal

Reply:
I hear ya. A part of me likes the consistency of the `case/when` syntax, particularly for people new to the repo that are just looking for a "copy/paste" of something that's _similar_ to what they need to do. If these types of conditions follow the same format then it reduces some cognitive effort for a newbie to transform from the `==` to knowing they need either an `if/else` or a `case/when` and can see, "oh, I just need to add another `when` condition.

At least that's the fairy tale I tell myself